Sources/PointFree/Episode/Show.swift

```diff
@@ -38,7 +38,125 @@ let episodeResponse: Middleware<StatusLineOpen, ResponseEnded, Tuple4<Either<Str
     }
 )
 
-let episodeView = View<(Database.User?, Stripe.Subscription.Status?, Episode)> { user, subscriptionStatus, episode in
+let useCreditResponse =
+  filterMap(
+    over1(episode(forParam:)) >>> require1 >>> pure,
+    or: writeStatus(.notFound) >-> respond(episodeNotFoundView.contramap(lower))
+    )
+    <<< { userEpisodePermission >-> $0 } // Is there a name for this and/or do we want to introduce someting like "lift" to do it?
```

Comment:
this is kinda interesting. `userEpisodePermission` is just a plain old middleware, but we can easily lift it into a transformer by doing `{ userEpisodePermission >-> $0 }`. wonder if we should give that a name.

Reply:
That's neat! Will have to think on it. Maybe we wait for more use cases?